### PR TITLE
Release 1.10.0 disable tests

### PR DESCRIPTION
**Pull request by jsheng-jian**
```
Running tests...
Test project /home/ubuntu/neo-ai/neo-ai-dlr/build
      Start  1: dlr_allocator_test
 1/14 Test  #1: dlr_allocator_test ...............   Passed    0.97 sec
      Start  2: dlr_common_test
 2/14 Test  #2: dlr_common_test ..................   Passed    0.00 sec
      Start  3: dlr_data_transform_test
 3/14 Test  #3: dlr_data_transform_test ..........   Passed    0.02 sec
      Start  4: dlr_elem_test
 4/14 Test  #4: dlr_elem_test ....................   Passed    4.49 sec
      Start  5: dlr_pipeline_skl_xgb_test
 5/14 Test  #5: dlr_pipeline_skl_xgb_test ........   Passed    0.00 sec
      Start  6: dlr_pipeline_test
 6/14 Test  #6: dlr_pipeline_test ................   Passed    0.01 sec
      Start  7: dlr_relayvm_elem_test
 7/14 Test  #7: dlr_relayvm_elem_test ............   Passed    2.37 sec
      Start  8: dlr_relayvm_test
 8/14 Test  #8: dlr_relayvm_test .................   Passed    1.52 sec
      Start  9: dlr_test
 9/14 Test  #9: dlr_test .........................   Passed    4.86 sec
      Start 10: dlr_treelite_test
10/14 Test #10: dlr_treelite_test ................   Passed    0.02 sec
      Start 11: dlr_tvm_elem_test
11/14 Test #11: dlr_tvm_elem_test ................   Passed    2.36 sec
      Start 12: dlr_tvm_test
12/14 Test #12: dlr_tvm_test .....................   Passed    2.66 sec
      Start 13: dlr_dlsym_test
13/14 Test #13: dlr_dlsym_test ...................   Passed    4.16 sec
      Start 14: dlr_multiple_lib_test
14/14 Test #14: dlr_multiple_lib_test ............   Passed    0.87 sec

100% tests passed, 0 tests failed out of 14

Total Test time (real) =  24.33 sec
```

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -456,13 +456,13 @@ if(DLR_BUILD_TESTS AND NOT(AAR_BUILD))
...
+      /tmp/xgboost-ml_m5.tar.gz
       SHA1
-      4c8ac5f20db6c7c6d674dd2ac9d9e14ce887281e
+      1e45bb9d6108d70ac4ff37855cf13061d61ef742
     )
     # this is OS-agnostic
-    execute_process(COMMAND ${CMAKE_COMMAND} -E tar -xf /tmp/xgboost_test.tar.gz
+    execute_process(COMMAND ${CMAKE_COMMAND} -E tar -xf /tmp/xgboost-ml_m5.tar.gz
                     WORKING_DIRECTORY ${XGBOOST_TEST_MODEL})
     file(REMOVE /tmp/xgboost_test.tar.gz)
```
Comment (apivovarov) [inline CMakeLists.txt:467]: Should we remove `/tmp/xgboost-ml_m5.tar.gz` now?

File: tests/cpp/dlr_pipeline_skl_xgb_test.cc [hunk 2]
```diff
@@ -156,5 +156,7 @@ int main(int argc, char** argv) {
 #ifndef _WIN32
   testing::FLAGS_gtest_death_test_style = "threadsafe";
 #endif  // _WIN32
+  // temporarily disable tests (TO DO: generate new artifacts)
```
Comment (apivovarov) [inline tests/cpp/dlr_pipeline_skl_xgb_test.cc:159]: Lets use `TODO` (without space in the middle) - so we can easily find all TODOs if we search for TODO

File: tests/cpp/dlr_data_transform_test.cc [hunk 3]
```diff
@@ -225,7 +225,7 @@ TEST(DLR, DataTransformDateTime) {
   }
 }
 
-TEST(DLR, RelayVMDataTransformInput) {
+TEST(DLR, DISABLED_RelayVMDataTransformInput) {
```
Comment (apivovarov) [inline tests/cpp/dlr_data_transform_test.cc:228]: Lets add a reason, recommendations and TODO label in a new comment above

File: CMakeLists.txt (hunk 1 above)
Comment (apivovarov) [reply CMakeLists.txt:467]: I missed this in your prev PR. BTW, you need to rebase this PR because The changes above were already merged to 1.10.0

Comment (jsheng-jian) [reply CMakeLists.txt]: Thanks, updated and rebased the commit

File: tests/cpp/dlr_pipeline_skl_xgb_test.cc (hunk 2 above)
Comment (jsheng-jian) [reply tests/cpp/dlr_pipeline_skl_xgb_test.cc]: Done

File: tests/cpp/dlr_data_transform_test.cc (hunk 3 above)
Comment (jsheng-jian) [reply tests/cpp/dlr_data_transform_test.cc:229]: Done

**Review by apivovarov — APPROVED** [review]

**Pull request merged**